source/common/tracing/zipkin/tracer.h

```diff
@@ -15,12 +15,12 @@ namespace Zipkin {
  * of "reporting" a Zipkin span that has ended its life cycle. "Reporting" can mean that the
  * span will be sent to out to Zipkin, or buffered so that it can be sent out later.
  */
-class Reporter {
+class ReporterInterface {
```

Comment:
To match the rest of the code, please leave this as Reporter, and make the implementation class ReporterImpl